sdk/storage/Azure.Storage.Blobs/tests/BlobBaseClientTests.cs

```diff
@@ -429,6 +429,34 @@ async Task Verify(Response<BlobDownloadInfo> response)
                 TestHelper.AssertSequenceEqual(data, actual.ToArray());
             }
         }
+
+        [Test]
+        public async Task DownloadAsync_ObjectReplication()
+        {
+            // This is a recorded ONLY test with a special container we previously setup, as we can't auto setup policies yet
+            var source_container = InstrumentClient(new BlobContainerClient("<connection_string>", "test1"));
+            var dest_container = InstrumentClient(new BlobContainerClient("<connection_string>", "test2"));
+
+            string blob_name = GetNewBlobName();
+            var source_blob = source_container.GetBlobClient(blob_name);
+            var dest_blob = dest_container.GetBlobClient(blob_name);
+            try
+            {
+                await source_blob.UploadAsync(new MemoryStream(GetRandomBuffer(Constants.KB)));
+                await Task.Delay(300000);
```

Comment:
What are we waiting for?  Also, Ted prefers we use `https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Common/tests/Shared/StorageTestBase.cs#L338`

Reply:
It takes at least 5 minutes for the replication to occur. If we try to get the properties, it will not show up. (I will use add the delay instead in the next commit)

Reply:
What I ended up doing was just hard coding a blob name and only doing getProperties on it instead of uploading. Since it's only recorded, it'll always be the same and not have any issues with collisions, and then you don't have to wait at all.